src/main/scala/com.snowplowanalytics/forex/Forex.scala

```diff
@@ -132,13 +138,13 @@ case class ForexLookupTo[F[_]](
  * @param config Forex config
  * @param client Passed down client that does actual work
  */
-case class ForexLookupWhen[F[_]: Sync](
+final case class ForexLookupWhen[F[_]: Monad](
```

Comment:
It is a part of tagless final convention that data types are unconstrained and all minimal constraints imposed only on functions.

Reply:
I removed the constraints from the types and put them in the methods that needed them. However, I couldn't do it in the original commit, where they were introduced because there have been many changes since then and the rebase was a nightmare. I put them as close to the end as possible: bf01f68bf8803912e25db987a185222267a177dd.